src/bin/common/solving_loop.ml

```diff
@@ -496,6 +506,7 @@ let main () =
         sat;
       st
     | Tableaux | Tableaux_CDCL | CDCL | CDCL_Tableaux ->
+      O.set_sat_solver sat;
```

Comment:
Is it necessary to add this back?

Reply:
This, like the previous one, are rebase artifacts; thanks for pointing them out.